### PR TITLE
Add Token Requestor client :sparkles:

### PR DESCRIPTION
**Pull request by junnae**

### CODE REVIEW

File: java/token-requestor-client/src/main/java/no/bankaxept/epayment/client/tokenrequestor/TokenRequestorClient.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        } catch (JsonProcessingException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    private <T> T deserializeOrNull(Class<T> clazz, String input) {
+        try {
+            return objectMapper.readValue(input, clazz);
+        } catch (JsonProcessingException e) {
+            return null;
```
Comment (thxmasj) [inline java/token-requestor-client/src/main/java/no/bankaxept/epayment/client/tokenrequestor/TokenRequestorClient.java:56]: swallowed exception

File: java/base-client/src/main/java/no/bankaxept/epayment/client/base/RequestResponse.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
+package no.bankaxept.epayment.client.base;
+
+import java.util.Objects;
+
+public class RequestResponse<T> {
+    private final RequestStatus status;
+    private final T response;
```
Comment (adrianhj88) [inline java/base-client/src/main/java/no/bankaxept/epayment/client/base/RequestResponse.java:7]: I think RequestResponse sounds a bit weird. But I also understand why it is named like that. Could it however be named like Response, with Status status and T body? Or something similar?

File: java/token-requestor-client/src/main/java/no/bankaxept/epayment/client/tokenrequestor/TokenRequestorClient.java [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+    public TokenRequestorClient(BaseClient baseClient) {
+        this.baseClient = baseClient;
+    }
+
+    public TokenRequestorClient(String baseurl, String apimKey, String username, String password) {
+        this.baseClient = new BaseClient(baseurl, apimKey, username, password);
+    }
+
+    public Flow.Publisher<RequestResponse<EnrolCardResponse>> enrol(EnrolCardRequest request, String correlationId) {
```
Comment (thxmasj) [inline java/token-requestor-client/src/main/java/no/bankaxept/epayment/client/tokenrequestor/TokenRequestorClient.java:33]: There should be no body in the response, so we should fix that first. Then you don't need the `RequestResponse`. We did a similar change on the Merchant API not long ago, but we must have forgotten to do it on this one.

**Review by thxmasj — CHANGES_REQUESTED** [review]

File: java/base-client/src/main/java/no/bankaxept/epayment/client/base/RequestResponse.java (hunk 2 above)
Comment (thxmasj) [reply java/base-client/src/main/java/no/bankaxept/epayment/client/base/RequestResponse.java:7]: We don't need the class at all, re. my [other comment](https://github.com/techcloud0/bankaxept-epayment-development-kit/pull/16#discussion_r847165606).

File: java/token-requestor-client/src/main/java/no/bankaxept/epayment/client/tokenrequestor/TokenRequestorClient.java (hunk 3 above)
Comment (junnae) [reply java/token-requestor-client/src/main/java/no/bankaxept/epayment/client/tokenrequestor/TokenRequestorClient.java]: Is it never gonna be needed? Removed it now

File: java/token-requestor-client/src/main/java/no/bankaxept/epayment/client/tokenrequestor/TokenRequestorClient.java (hunk 1 above)
Comment (junnae) [reply java/token-requestor-client/src/main/java/no/bankaxept/epayment/client/tokenrequestor/TokenRequestorClient.java]: This is removed now

File: java/token-requestor-client/src/main/java/no/bankaxept/epayment/client/tokenrequestor/TokenRequestorClient.java (hunk 3 above)
Comment (junnae) [reply java/token-requestor-client/src/main/java/no/bankaxept/epayment/client/tokenrequestor/TokenRequestorClient.java]: Is the tokenId supplied in a callback now or something?

Comment (thxmasj) [reply java/token-requestor-client/src/main/java/no/bankaxept/epayment/client/tokenrequestor/TokenRequestorClient.java]: Yes, there should be a callback.

Comment (thxmasj) [reply java/token-requestor-client/src/main/java/no/bankaxept/epayment/client/tokenrequestor/TokenRequestorClient.java]: `components.schemas.EnrolCardCallbackData` should have a tokenId similar to `components.schemas.PaymentRequestCallbackData` having a paymentId.

Comment (junnae) [reply java/token-requestor-client/src/main/java/no/bankaxept/epayment/client/tokenrequestor/TokenRequestorClient.java]: I created an [issue](https://github.com/techcloud0/bankaxept-epayment-development-kit/issues/17) to update this

**Review by thxmasj — APPROVED** [review]

**Pull request merged**